search/shortest_test.go

```diff
@@ -0,0 +1,500 @@
+// Copyright ©2014 The gonum Authors. All rights reserved.
```

Comment:
This one is correct, because the tests enumerated here are derived from the original 2014 file.